cosy.py

```diff
@@ -337,6 +338,32 @@ def check_completeness(symbols):
         print("Your output will be incomplete!")
 
 
+def demangle(symtable):
+    """Replace the alias entries with more readable names for the same symbol in-place
+
+    This is done by passing the collected symbol names through the ``rustfilt``
+    program if it is present, and returning without any actions otherwise.
+    """
+
+    all_symbols = list(itertools.chain(*([s['sym']] + s['alias'] for s in symtable)))
+    assert not any("\n" in s for s in all_symbols)
+    symbol_count = len(all_symbols)
+    all_symbols = "\n".join(all_symbols)
+
+    try:
+        filtered_symbols = subprocess.check_output(['rustfilt'], input=all_symbols.encode('utf8'))
+    except FileNotFoundError:
+        print("Warning: No `rustfilt` program found, symbols will not be demangled", file=sys.stderr)
```

Comment:
```suggestion
        print("Warning: No `rustfilt` program found, Rust symbols will not be demangled", file=sys.stderr)
```

makes the warning a bit more precise and let us not care about it when no actual Rust code is in use

Reply:
I deliberately kept that general because C++ can also generate mangled symbols, and rustfilt claims to demangle them just as well.

Reply:
Ah, that wasn't clear to me, especially given the comment below about potential future addition of `c++filt`. Then let's leave as is.